edit-post/assets/stylesheets/main.scss

```diff
@@ -270,12 +270,3 @@ body.gutenberg-editor-page {
 .ephox-snooker-resizer-bar-dragging {
 	background: $blue-medium-500;
 }
-
-// This style is defined here instead of the modal component
```

Comment:
I'm not exactly sure what this means — the height of the modal is defined at responsive breakpoints already, this did not seem to account for that. But regardless of that, I may have been able to refactor this out by removing the need for `calc` here. Can you take a look, @youknowriad?

Reply:
It means we shouldn't assume we're in the context of the WP Admin page when tweaking the style of the modal in the `style.scss` of the modal component. Which means we can't use things like:

 - `$admin-bar-height`
 -  `body.is-fullscreen-mode` 

And as far as I can tell, this PR rewrites the modal component to be shown in the same way regardless of the pagee it's used in (editor or not), so I think we're probably fine removing those.